compose/config/config.py

```diff
@@ -450,9 +450,10 @@ def format_device_option(entity_type, config):
     device = config['driver_opts'].get('device')
     if o and o == 'bind' and device:
         fullpath = os.path.abspath(os.path.expanduser(device))
-        if not os.path.exists(fullpath):
-            raise ConfigurationError(
-                "Device path {} does not exist.".format(fullpath))
+        # we may want to enforce the path check later
+        # if not os.path.exists(fullpath):
+        #    raise ConfigurationError(
+        #        "Device path {} does not exist.".format(fullpath))
```

Comment:
Remove?

Reply:
why not check if docker_host is local?

Reply:
We do a check for the `local` driver a few lines above that but apart from that I think it is better to leave the docker engine to manage it to keep the old behaviour.